src/screens/Home/Home.js

```diff
@@ -138,6 +146,13 @@ function Home() {
     dispatch(fetchAllAccountsTotalBalancesAction());
   };
 
+  const onApps = async () => {
+    const storage = Storage.getInstance('db');
+    const nativeIntegrationWarning = await storage.get('ni_warning');
```

Comment:
Do we have a helper for this sort of thing? Just strange too see a direct accessing of the Storage instance...

Reply:
Okay, I'll add it in separate file.